drivers/i2c/i2c_npcx_port.c

```diff
@@ -108,6 +108,27 @@ static int i2c_npcx_port_transfer(const struct device *dev,
 	return ret;
 }
 
+#ifdef CONFIG_I2C_CONFIG_GET
+static int i2c_npcx_port_config_get(const struct device *dev, uint32_t *dev_config)
+{
+	const struct i2c_npcx_port_config *const config = DRV_CONFIG(dev);
+	uint32_t speed;
+	int ret;
+
+	if (config->i2c_ctrl == NULL) {
+		LOG_ERR("Cannot find i2c controller on port%02x!", config->port);
+		return -EIO;
+	}
+
+	ret = npcx_i2c_ctrl_get_speed(config->i2c_ctrl, &speed);
+	if (!ret) {
+		*dev_config = (I2C_MODE_MASTER | I2C_ADDR_10_BITS | speed);
```

Comment:
Don't support I2C_ADDR_10_BITS

Reply:
Done.

Reply:
The commit log for 10-bits might be confused. Let's fix it.

Reply:
Yes, thanks!